app/medInria/medFilteringWorkspace.cpp

```diff
@@ -44,6 +44,7 @@ medFilteringWorkspace::medFilteringWorkspace(QWidget *parent): medAbstractWorksp
 {
     d->filteringToolBox = new medFilteringSelectorToolBox(parent);
     connect(d->filteringToolBox,SIGNAL(processFinished()),this,SLOT(onProcessSuccess()));
+    d->filteringToolBox->setWorkspace(this);
```

Comment:
Yep, i needed it on the first version of this PR :)
This is just a missing code that i found.